lib/LTP/utils.pm

```diff
@@ -87,6 +88,23 @@ sub get_ltp_version_file {
     return get_ltproot($want_32bit) . '/version';
 }
 
+sub log_versions {
+    my $kernel_pkg     = is_jeos ? 'kernel-default-base' : 'kernel-default';
+    my $kernel_pkg_log = '/tmp/kernel-pkg.txt';
+
+    upload_logs('/boot/config-$(uname -r)', failok => 1);
+
+    script_run("rpm -qi $kernel_pkg > $kernel_pkg_log 2>&1");
+    upload_logs($kernel_pkg_log, failok => 1);
+
+    record_info('KERNEL VERSION',     script_output('uname -a'));
+    record_info('KERNEL DEFAULT PKG', script_output("cat $kernel_pkg_log",          proceed_on_failure => 1));
+    record_info('KERNEL EXTRA PKG',   script_output('rpm -qi kernel-default-extra', proceed_on_failure => 1));
+    record_info('KERNEL CONFIG',      script_output('cat /boot/config-$(uname -r)'));
+    record_info('ver_linux',          script_output(get_ltproot . '/ver_linux'));
```

Comment:
Since you call `ver_linux` here, you should remove the duplicate call from `init_ltp_tests()`.